tidb-cloud/migrate-from-mysql-using-data-migration.md

```diff
@@ -145,7 +145,14 @@ If your MySQL service is in a Google Cloud VPC, take the following steps:
 
 ### Enable binary logs
 
-To perform incremental data migration, make sure you have enabled binary logs of the upstream database, and the binary logs have been kept for more than 24 hours.
+To perform incremental data migration, make sure the following requirements are met:
+
+- Binary logs are enabled for the upstream database.
+- The binary logs are retained for at least 24 hours.
+- The binlog format for the upstream database is set to `ROW`. If not, update the format to `ROW` as follows to avoid the [format error](/tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md#error-message-check-whether-mysql-binlog_format-is-row):
+
+    - MySQL: execute the `set global binlog_format=ROW;` statement.
```

Comment:
```suggestion
    - MySQL: execute the `SET GLOBAL binlog_format=ROW;` statement.
```

What about `SET PERSIST binlog_format=ROW;` to make this persist across reboots?